Resources/doc/index.md

```diff
@@ -40,7 +40,7 @@ public function registerBundles()
 
 ### Step 3: Register the routing definition
 
-``` yml
```

Comment:
just to make it consistent to be either yml everywhere or yaml which is more used and correct